relay-server/src/actors/processor.rs

```diff
@@ -2283,6 +2283,8 @@ impl EnvelopeProcessorService {
             .project_state
             .has_feature(Feature::SpanMetricsExtractionAllModules);
 
+        let op_denylist = vec!["clickhouse", "mongodb", "redis", "activerecord"];
```

Comment:
can this be a set? I think we can avoid the clone into iter boilerplate then below.